### PR TITLE
Removed unruly AB test

### PR DESCRIPTION
**Pull request by jeteve**
## What does this change?
This PR removes unruly AB test as we're happy with it, according to the card:

https://trello.com/c/w2ql7bpp/148-remove-unruly-a-b-test

### CODE REVIEW

**Review by kelvin-chappell — APPROVED** [review]
👍

**Comment by JonNorman** [comment]
Do we still want this to be behind a feature switch (as opposed to an AB switch)? @ScottPainterGNM

File: static/src/javascripts/projects/common/modules/experiments/ab-tests.js [hunk 1]
```diff
@@ -31,6 +29,7 @@ export const getActiveTests = (): $ReadOnlyArray<ABTest> =>
 export const getExpiredTests = (): $ReadOnlyArray<ABTest> =>
     TESTS.filter(test => isExpired(test.expiry));
 
+// UnrulyPerformanceTest
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/experiments/ab-tests.js]: What's this comment for?

**Comment by ScottPainterGNM** [comment]
@JonNorman I think not, this actually wasn't controlling Unruly being on or off anyway, just keeping a portion of the audience in a test for monitoring purposes. So think this is good to go. If we want to switch off unruly in the future I think we can just handle that via the line items.

**Pull request merged**